app/models/user.rb

```diff
@@ -82,12 +82,18 @@ class User
   # email presence validated through Devise
   validates :name, presence: true
   validates :role, presence: true
-  validate :secure_password
+  validate :secure_password, if: :updating_password?
+  # i18n-tasks-use t('errors.messages.password.password_strength')
+  validates :password, password_strength: {use_dictionary: true}, if: :updating_password?
+
 
   TIME_BEFORE_DISABLED_BY_FUND = 9.months
 
+  def updating_password?
```

Comment:
I think there's a nicer pg magic method we'll be able to use when we switch over; we'll have to check